v6.8.x build fails with 'errno' was not declared in this scope

XCSoar versions having and not having the problem
-------------------------------------------------

v6.8.x has the problem, master and v7.0_preview15 apparently don't suffer

Expected behavior
-----------------

Build with target Kobo succeeds without errors.

Actual behavior
---------------

There is a compilation error as below, which can be addressed by bringing in the change from https://github.com/nico-krais/XCSoar/commit/d1789700cc5806d66b38ece6c67de171fd202326

```
  CXX     output/KOBO/opt/src/Event/Poll/Linux/Input.o
src/Event/Poll/Linux/Input.cpp: In member function 'void LinuxInputDevice::Read()':
src/Event/Poll/Linux/Input.cpp:133:9: error: 'errno' was not declared in this scope
     if (errno != EAGAIN && errno != EINTR)
         ^~~~~
src/Event/Poll/Linux/Input.cpp:133:9: note: 'errno' is defined in header '<cerrno>'; did you forget to '#include <cerrno>'?
src/Event/Poll/Linux/Input.cpp:35:1:
+#include <cerrno>
 #include <sys/ioctl.h>
src/Event/Poll/Linux/Input.cpp:133:9:
     if (errno != EAGAIN && errno != EINTR)
         ^~~~~
src/Event/Poll/Linux/Input.cpp:133:18: error: 'EAGAIN' was not declared in this scope
     if (errno != EAGAIN && errno != EINTR)
                  ^~~~~~
src/Event/Poll/Linux/Input.cpp:133:18: note: suggested alternative: 'KEY_AGAIN'
     if (errno != EAGAIN && errno != EINTR)
                  ^~~~~~
                  KEY_AGAIN
src/Event/Poll/Linux/Input.cpp:133:37: error: 'EINTR' was not declared in this scope
     if (errno != EAGAIN && errno != EINTR)
                                     ^~~~~
src/Event/Poll/Linux/Input.cpp:133:37: note: suggested alternative: 'VINTR'
     if (errno != EAGAIN && errno != EINTR)
                                     ^~~~~
                                     VINTR
make: *** [build/compile.mk:108: output/KOBO/opt/src/Event/Poll/Linux/Input.o] Error 1
```


Steps to reproduce the behavior
-------------------------------

Build the docker image as per https://github.com/XCSoar/XCSoar/tree/master/ide/docker and attempt to run the following steps to build the KoboRoot.tgz

```
make clean
make TARGET=KOBO DEBUG=n USE_CCACHE=y kobo-libs
make TARGET=KOBO DEBUG=n USE_CCACHE=y
make TARGET=KOBO DEBUG=n USE_CCACHE=y output/KOBO/KoboRoot.tgz
```

Do you have any idea what may have caused this?
-----------------------------------------------

There is some discussion in https://github.com/XCSoar/XCSoar/issues/197 but even with the apparent fix backported since v6.8.13 my own build of v6.8.15 and v6.8.x seemed to be failing with the same error as before the backport. Adding the one line change proposed by @nico-krais made it possible for me to build for Kobo.

Do you have an idea how to solve the issue?
-------------------------------------------

I am raising a PR for the change which fixed it for me.

--- a/src/Event/Poll/Linux/Input.cpp
+++ b/src/Event/Poll/Linux/Input.cpp
@@ -33,6 +33,7 @@ Copyright_License {
 
 #include <termios.h>
 #include <sys/ioctl.h>
+#include <errno.h>
 
 template<typename T>
 static constexpr unsigned

